src/interfaces/superjson/superjson.utils.ts

```diff
@@ -11,6 +10,10 @@ import {
   SEMVER_REGEX,
   SuperJsonDocument,
 } from './superjson';
+import * as schema from './superjson.schema.json';
+
+const prepareIs = preparePrepareIs(schema);
+const assertEquals: Assert<SuperJsonDocument> = prepareAssert(schema);
```

Comment:
I'm kinda sad you named all of these `assertEqual` because it's confusing to read, but I guess whatever.